### PR TITLE
[1.3] Add observability documentation

### PR DESCRIPTION
**Pull request by eugenesk24**
### Description
Adds observability documentation on:
- Application Analytics
- Event Analytics
  - View logs
    - Trace log correlation
    - Surrounding events
    - Live tail
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _observability-plugin/event-analytics.md [hunk 1]
```diff
@@ -30,4 +30,27 @@ For more information about building PPL queries, see [Piped Processing Language]
...
-To save a visualization, expand the save dropdown menu next to **Run**, enter a name for your visualization, then choose **Save**. You can reopen any saved visualizations on the event analytics page.
+To save a visualization, expand the save dropdown menu next to **Refresh**, enter a name for your visualization, then choose **Save**. You can reopen any saved visualizations on the event analytics page.
+
+## View logs
+
+Users have several ways to view their logs.
+
+### Trace log correlation
+
+Users who regularly track events across applications have the capability to correlate Logs and Traces. To view the correlation, users have to index the traces as per the Open Telemetry standards (similar to trace analytics). Once users add a TraceId field to their logs, they can view the correlated trace information in the event explorer log details.  This will allow users to correlate logs and traces that correspond to the same execution context.
```
Comment (keithhc2) [inline _observability-plugin/event-analytics.md:41]: Can you provide some context about what it means to correlate logs and traces?

File: _observability-plugin/app-analytics.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Select services & entities from the dropdown or the service map.
+- Select trace groups from the dropdown or the table.
+1. Choose **Create**.
+
+### Create a visualization
+
+1. Choose the **Log Events** tab
+1. Use [PPL]({{site.url}}{{site.baseurl}}/observability-plugin/ppl/index) to build upon your base query
+1. Move to the **Visualizations** tab to see your visualizations
+1. Expand the **Save** dropdown menu, enter a nem for your visualization, then choose **Save**.
```
Comment (keithhc2) [inline _observability-plugin/app-analytics.md:34]: Typo of "nem" here. I'm assuming this is supposed to be name?

File: _observability-plugin/app-analytics.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+To see your visualizations, move to the **Panel** tab. 
+
+### Configure availability
+
+In order to configure availability, you must add availability levels to a line type time series metric. Users can use the power of the PPL language to define arbitrary conditions on their logs to create a time series metric.
```
Comment (keithhc2) [inline _observability-plugin/app-analytics.md:40]: Is there something we can link off to for "line type time series metric" here? Also, some context about what availability is might be useful here.

**Review by keithhc2 — CHANGES_REQUESTED** [review]
Overall looks pretty good. There are some language tweaks I want to make, but that can happen after we merge in the PR

File: _observability-plugin/app-analytics.md (hunk 2 above)
Comment (eugenesk24) [reply _observability-plugin/app-analytics.md:34]: 😆  yes it is

**Review by keithhc2 — APPROVED** [review]
@Naarcha-AWS Content looks fine to me here. I'll make some language tweaks after we merge.

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**